src/include/access/rmgrlist.h

```diff
@@ -47,3 +47,8 @@ PG_RMGR(RM_COMMIT_TS_ID, "CommitTs", commit_ts_redo, commit_ts_desc, commit_ts_i
 PG_RMGR(RM_REPLORIGIN_ID, "ReplicationOrigin", replorigin_redo, replorigin_desc, replorigin_identify, NULL, NULL, NULL)
 PG_RMGR(RM_GENERIC_ID, "Generic", generic_redo, generic_desc, generic_identify, NULL, NULL, generic_mask)
 PG_RMGR(RM_LOGICALMSG_ID, "LogicalMessage", logicalmsg_redo, logicalmsg_desc, logicalmsg_identify, NULL, NULL, NULL)
+PG_RMGR(RM_RESERVED1_ID, "reserved", NULL, NULL, NULL, NULL, NULL, NULL)
+PG_RMGR(RM_RESERVED2_ID, "reserved", NULL, NULL, NULL, NULL, NULL, NULL)
+PG_RMGR(RM_RESERVED3_ID, "reserved", NULL, NULL, NULL, NULL, NULL, NULL)
+PG_RMGR(RM_RESERVED4_ID, "reserved", NULL, NULL, NULL, NULL, NULL, NULL)
```

Comment:
Why these reserved IDs?

Reply:
before [1] it served as a protection from new resource managers introduced by the community (it's rather fake protection now as we don't have persisted customer data yet).
now we may need to decide if pulling [2] into Neon would make it cleaner

[1] https://www.postgresql.org/message-id/flat/62d88a73bcc3ba869d5d5e12c08145be8c951d37.camel%40j-davis.com#d731a8a0c9e259a11fd8376247adced5
[2] https://github.com/postgres/postgres/commit/5c279a6d350205cc98f91fb8e1d3e4442a6b25d1